src/dql.erl

```diff
@@ -1,8 +1,69 @@
 -module(dql).
--export([prepare/1, parse/1, unparse/1, glob_match/2]).
+-export([prepare/1, parse/1, unparse/1, glob_match/2, flatten/1, unparse_metric/1]).
```

Comment:
Line 2 is too long: -export([prepare/1, parse/1, unparse/1, glob_match/2, flatten/1, unparse_metric/1])..